src/System.Drawing.Common/tests/ColorTranslatorTests.cs

```diff
@@ -188,7 +190,16 @@ public static IEnumerable<object[]> FromHtml_TestData()
         [MemberData(nameof(FromHtml_TestData))]
         public void FromHtml_String_ReturnsExpected(string htmlColor, Color expected)
         {
-            Assert.Equal(expected, ColorTranslator.FromHtml(htmlColor));
+            CultureInfo originalCulture = Thread.CurrentThread.CurrentCulture;
```

Comment:
We have this test class: https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/ThreadCultureChange.cs

Might be worth adding a ctor for the culture to it